code/data_structures/src/stack/Stack using Linked-Lists.cpp

```diff
@@ -0,0 +1,75 @@
+#include<iostream>
+using namespace std;
+
+template <typename T>
+
+class Node {
+    public :
+    T data;
+    Node<T> *next;
+    
+    Node(T data) {
+        this -> data = data;
+        next = NULL;
+    }
+};
+
+class Stack {
+    Node<T> *head;
+    	
+    int size;							// number of elements prsent in stack
+    
+    public :
+    
+    Stack() {							//Constructor
+      head = NULL; 
+      size=0;
+    }
```

Comment:
Consider adding another constructor that accepts a `std::initializer_list` and initializes the stack with said elements.